src/core/net/ip6.hpp

```diff
@@ -93,6 +94,8 @@ using ot::Encoding::BigEndian::HostSwap32;
  *
  */
 
+#define MAKE_DIVIDABLE_BY_EIGHT(length) (length) & 0xfff8
```

Comment:
nit: rename to `MAKE_DIVISIBLE_BY_EIGHT`?

Reply:
yes, good idea. changed.